lisa/schema.py

```diff
@@ -430,6 +430,10 @@ class DiskOptionSettings(FeatureSettings):
         default=None,
         metadata=metadata(allow_none=True, decoder=search_space.decode_count_space),
     )
+    nvme_disk_count: search_space.CountSpace = field(
```

Comment:
My understanding is, the nvme is a independent feature, like serial console. Because it's enabled in some of vm_size only. It doesn't make sense to add it in disk options.